### ISSUE TITLE
Provide --help

### ISSUE THREAD

**Issue by JakobDev** [Heroic-Games-Launcher/HeroicGamesLauncher#1638]
### Problem description

Heric has commandline options such as `--no-gui` or `--fullscreen`. You can also direct start a Game with the cli. But heroic has no `--help` or `-h` switch. If I want to know what cli options a Program provides I usually run the program with `--help`. This usually prints just all cli options don't start the program. It is supported by almost every program which has cli options.

### Feature description

When the User passes `--help` or `-h`, Heroic should just print all cli options and then exit, like any other program.

You could use a package to archive that. After a short search I found [argparse](https://www.npmjs.com/package/argparse), which provides a cli parser and has a built-in `--help` and `-h` switch. That would be far better than the current method and would make it easier to add more cli options in the future.

**Comment by flavioislima** [comment]
Yes we are considering using this package but we need to look if Electron already has a support for something like this.

**Comment by sonofevil** [comment]
It is weird that heroic doesn't have this. I have scoured the documentation for explanations of the options but there are none. I get that this is a GUI-focused app but this is just bad practice on Linux. Please provide *some* documentation of the CLI options. Thanks.
